src/pixelator/report/qcreport/main.py

```diff
@@ -51,18 +51,20 @@ def create_dynamic_report(
 
     # Collect antibody metrics
     antibodies_data_values = {
-        "antibody_reads": demux_metrics.output_read_count,
+        "antibody_reads": reads_flow.valid_antibody_read_count,
         "antibody_reads_usable_per_cell": annotate_metrics.read_count_per_cell_stats.mean,
         "antibody_reads_in_outliers": annotate_metrics.reads_in_aggregates_count,
         "unrecognized_antibodies": demux_metrics.unrecognised_antibody_read_count,
     }
 
-    antibodies_data_fractions = (
-        pd.Series(antibodies_data_values) / reads_flow.input_read_count
-    ).to_dict()
-
+    total_reads_per_cell = reads_flow.input_read_count / annotate_metrics.cell_count
     antibodies_data_fractions = {
-        f"fraction_{k}": v for k, v in antibodies_data_fractions.items()
+        "fraction_antibody_reads": reads_flow.fraction_valid_antibody_reads,
+        "fraction_antibody_reads_usable_per_cell": (
+            annotate_metrics.read_count_per_cell_stats.mean / total_reads_per_cell
```

Comment:
Will this solve TECH-68 too? If that is the case, include it in the PR and link to that ticket too.

Reply:
Yes, it should. I noticed this bug while working on it and forgot there was already a ticket for it.
I have linked it.